src/windows/CameraProxy.js

```diff
@@ -696,21 +696,22 @@ function takePictureFromCameraWindows(successCallback, errorCallback, args) {
     // decide which max pixels should be supported by targetWidth or targetHeight.
     var maxRes = null;
     var UIMaxRes = WMCapture.CameraCaptureUIMaxPhotoResolution;
+    var totalPixels = targetWidth * targetHeight;
     switch (true) {
```

Comment:
As long as you're refactoring the numbers, let's also turn this from `switch (true)` into `if`/`else`.